app/src/main/java/com/belatrixsf/allstars/adapters/EmployeeListAdapter.java

```diff
@@ -46,12 +50,14 @@ public EmployeeViewHolder onCreateViewHolder(ViewGroup parent, int viewType) {
     public void onBindViewHolder(EmployeeViewHolder holder, int position) {
         Employee employee = employeeList.get(position);
         String fullName = employee.getFullName();
-        String currentFirstLetter = ( fullName != null && !fullName.isEmpty() ) ? String.valueOf(fullName.charAt(0)).toUpperCase() : "#";
+        String currentFirstLetter = ( fullName != null && !fullName.isEmpty() ) ? String.valueOf(fullName.charAt(CAP_POSITION)).toUpperCase() : NUMERIC_SYMBOL;
```

Comment:
As @gyosida said. Avoid using spaces between (_... Use command+shift+L to use the auto indenter for android